components/multi-platform-controller/staging/host-values.yaml

```diff
@@ -162,6 +162,8 @@ dynamicConfigs:
       chmod a+rw /etc/cdi/nvidia.yaml
       --//--
 
+  macos-mac2metal-arm64: {}
```

Comment:
one more thing, The setup was done in AWS internal staging so let's move this to staging-downstream

Reply:
I can do it, no need to do it now :)

Reply:
Done, stay in week end mode :)